src/agent/src/storage/mod.rs

```diff
@@ -380,8 +381,122 @@ fn mount_storage(logger: &Logger, storage: &Storage) -> Result<()> {
         "mount-options" => options.as_str(),
     );
 
+    let confidential = storage.driver_options.contains(&"confidential=true".into());
+    let ephemeral = storage.driver_options.contains(&"ephemeral=true".into());
```

Comment:
Would this work too?

contains("ephemeral=true".to_string());

That would be easier for me to read/understand. Is there some advantage of using into() here?

Reply:
Done